docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,183 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later works with Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection.  Coherence caches can be accessed from outside the Coherence cluster via Coherence*Extend, REST, and other supported Coherence clients.  
+
```

Comment:
Add something to make it clear that using Istio with Coherence and the Operator are separate. For example

Using Coherence clusters with Istio does not require the Coherence Operator to also be using Istio (and vice-versa) . The Coherence Operator can manage Coherence clusters independently of whether those clusters are using Istio.

Reply:
added